### DISCUSSION TITLE
컴퓨터 네트워크(승은-> 민수)

### DISCUSSION THREAD

**Discussion by sep037** [tempdku/temp#8] [General]
@skyriv213 

1. 네트워크 부하로 발생면 특정 지역에 (   A   )이 발생할 수도 있는데, 데이터 전송 경로와 관계가 있어 네트워크 계층이 제어를 한다. 이 A를 제어하기 위한 방법 중 1가지를 설명하시오 !

2. (   B   )는 네트워크 계층의 기능을 수행하는 IP 프로토콜이 호스트를 구분하기 위하여 사용하는 주소체계이다. 이 B는 몇 비트의 이진 숫자로 구성되는가 ?

3.  인터넷에서는 (   C   )프로토콜이 네트워크 계층의 기능을 수행하며, 연결형 서비스를 지원하는 (   D   ) 프로토콜과 (   E   ) 프로토콜이 전송 계층의 데이터 전송 기능을 수행한다. C,D,E를 순서대로 답하세요 ~

**Comment by skyriv213** [comment]
진짜...문제 에반데..

**Reply by skyriv213** [reply]
A : ?? - 찾아봄
혼잡 : 혼잡제어에는 슬로우 스타터라는 방식이 존재한다.해당 방식의 경우 데이터를 주고받을 수 있는 윈도우의 크기에 맞춰서 진행이 된다. Syn, Ack를 주고 받으면서 네트워크를 통신이 잘 이루어지면 전달하는 패킷의 갯수를 지수적으로 늘리는 것이다. 1,2,4,8,16 ..2^n승으로 진행이 되며 이때 혼잡이 발생되면 패킷의 전달 크기를 다시 1 혹은 혼잡이 발생한 크기의 절반부터 다시 전송을 시작한다.

B : IP 주소
IP주소의 경우 v4, v6가 존재한다. v4의 경우 10진수로 32비트로 표현을 했으며, v6의 경우 16진수로 128비트로 표현한다. 이때 v4의 한계가 다가오기에 이를 위한 대책으로 나온 것이 v6이다

C : IP
D : TCP
E : UDP

**Reply by sep037** [reply]
A 혼잡임
